librosa/filters.py

```diff
@@ -1115,7 +1124,7 @@ def window_bandwidth(window, n=1000):
 
     if key not in WINDOW_BANDWIDTHS:
         win = get_window(window, n)
-        WINDOW_BANDWIDTHS[key] = n * np.sum(win ** 2) / np.sum(np.abs(win)) ** 2
+        WINDOW_BANDWIDTHS[key] = n * np.sum(win ** 2) / (np.sum(win) ** 2 + util.tiny(win))
```

Comment:
Just a note that I'm adding a tiny here to stabilize the division.  This shouldn't affect anything substantively —I'd be worried if a window sums to an underflow value!—but it makes me feel better.